component/alertrules.libsonnet

```diff
@@ -64,6 +64,9 @@ local patch_alerts = {
   },
 };
 
+local runbook(alertname) =
+  'https://hub.syn.tools/rook-ceph/runbooks/%s.html' % alertname;
+
 /* FROM HERE: should be provided as library function by
```

Comment:
Is/will be there an ticket to have this converted to a library?

Reply:
https://github.com/appuio/component-openshift4-monitoring/issues/36